### PR TITLE
Update namespace in dev tools.

### PR DESCRIPTION
**Pull request by akatsoulas**

### CODE REVIEW

**Review by LOHAYBOY999 — CHANGES_REQUESTED** [review]
Boy999

File: k8s/templates/ingress.yaml
Comment (hossen71) [inline k8s/templates/ingress.yaml:1]: 25057d2185842ee2f5f85223e42065a241ae5f33

**Review by hossen71 — COMMENTED** [review]
good

File: k8s/values.yaml
Comment (hossen71) [inline k8s/values.yaml:1]: 25057d2185842ee2f5f85223e42065a241ae5f33

**Review by hossen71 — COMMENTED** [review]
25057d2185842ee2f5f85223e42065a241ae5f33

File: k8s/values.yaml [hunk 1]
```diff
@@ -1,4 +1,4 @@
 name: dino-park-dev-tools
-namespace: dino-park
+namespace: dinopark-dev
```
Comment (bheesham) [inline k8s/values.yaml:2]: Hm, this widens the gap between dev and prod. Though, I guess this is a double check for making sure we're in the right environment?

Are these expected to be ran in the same K8s cluster?

**Pull request merged**